views/index.ejs

```diff
@@ -1,10 +1,44 @@
 <% include partials/header %>
+<div class="FlagImgContainer"></div>
+<div class="RedBackgroundContainer">
```

Comment:
better name, you can use `ContentContainer` for now since I don't foresee any content living outside of this in our current design direction